src/Tools/AnalyzerRunner/App.config

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this has one problem. it let people to run analyzers with official roslyn bits, but not with dev bits (Debug or Release) which has 42.42.42.4242

since one of primary goal is letting community member to improve analyzer driver as well, I think we need to address that issue.

also, since one of goal of this is letting users to run IDE analyzer which is probably will be built with dev bits of compiler, we need to address that as well.

since this is currently part of Roslyn... can we do some magic? @mavasani @jasonmalinowski @tmat @jinujoseph @sharwell

Reply:
:arrow_right: We should just remove this and let the compiler automatically generate the necessary binding redirects.